windows/src/developer/TIKE/main/UframeBitmapEditor.pas

```diff
@@ -807,22 +812,44 @@ procedure TframeBitmapEditor.LoadFromFile(FileName: WideString);
   end;
 end;
 
+function IsPNGSignature(const buf: array of byte): Boolean;
+begin
+  Result :=
+    (buf[0] = $89) and
+    (buf[1] = $50) and
+    (buf[2] = $4E) and
+    (buf[3] = $47);
```

Comment:
What are these magic bytes?

Reply:
These are the PNG file signature, as the function is called ;-)